kong/plugins/request-transformer/access.lua

```diff
@@ -331,7 +335,9 @@ end
 local function transform_body(conf)
   local content_type_value = req_get_headers()[CONTENT_TYPE]
   local content_type = get_content_type(content_type_value)
-  if content_type == nil or #conf.rename.body < 1 and #conf.remove.body < 1 and #conf.replace.body < 1 and #conf.add.body < 1 and #conf.append.body < 1 then return end
+  if content_type == nil or #conf.rename.body < 1 and #conf.remove.body < 1 and #conf.replace.body < 1 and #conf.add.body < 1 and #conf.append.body < 1 then
```

Comment:
Again, a style update touching the conditions would - assuming we're not that zealous about commit atomicity - require that once touched, the conditions should match the required code style. We do get rid of the one-liner here but we should also get rid of the > 80 chars. Killing 2 birds at once like so would still be enclosed under the same scope of "improving the conditions code style".